### PR TITLE
Add bower

### LINKED ISSUE #12: Add component.json
Marionette supports component.json so if Metal is a dependency it will need to as well.

### PR DESCRIPTION
**Pull request by paulfalgout**
Resolves #12 and adds bower. Both of which `Mn` supports.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "test"
+  ],
+  "dependencies": {
+    "backbone": "^1.1.2",
+    "underscore": "^1.7.0"
+  }
+}
```
Comment (jamiebuilds) [inline bower.json]: NEEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWLLIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIINNNNNNNNNNNNNNNNNNNNNNNNNEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

Comment (paulfalgout) [inline bower.json]: heh.. I don't think my IDE helps me in `.json` extensions..   one sec

File: component.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "backbone-metal",
+  "version": "0.5.0",
+  "description": "Classes, Mixins, Errors, and more.",
+  "main": "./dist/backbone-metal.js",
+  "scripts": ["./dist/backbone-metal.js"],
+  "repo": "marionettejs/backbone-metal",
+  "license":  "ISC",
+  "licenses": [
```
Comment (jamiebuilds) [inline component.json]: Does it really need both of these?

**Comment by paulfalgout** [comment]
Removed component.json from this PR.  Will add in another for discussion, but this won't hold up bower

**Comment by jamesplease** [comment]
😱

**Comment by jamiebuilds** [comment]
Cool

**Pull request merged**